dask/dataframe/tests/test_rolling.py

```diff
@@ -91,9 +96,22 @@ def test_rolling_funtions_dataframe():
                     reason="rolling object not supported")
 def test_rolling_dataframe():
     df = pd.DataFrame({'a': np.random.randn(25).cumsum(),
-                       'b': np.random.randint(100, size=(25,))})
-    ddf = dd.from_pandas(df, 3)
-    basic_rolling_tests(df, ddf)
+                       'b': np.random.randint(100, size=(25,)),
+                       'c': np.random.randint(100, size=(25,)),
+                       'd': np.random.randint(100, size=(25,)),
+                       'e': np.random.randint(100, size=(25,))})
+    for n in [1, 2, 3]:
+        ddf = dd.from_pandas(df, n)
+
+        # Dataframe-specific tests (not shared with series)
+        eq(df.rolling(3, axis=1).max(), ddf.rolling(3, axis=1).max())
+        eq(df.rolling(3, axis='columns').mean(), ddf.rolling(3, axis='columns').mean())
+
+        eq(df.rolling(3, center=True, axis=1).max(), ddf.rolling(3, center=True, axis=1).max())
+        eq(df.rolling(3, center=False, axis=1).std(), ddf.rolling(3, center=False, axis=1).std())
+        eq(df.rolling(6, center=True, axis=1).var(), ddf.rolling(6, center=True, axis=1).var())
+        eq(df.rolling(7, center=True, axis='columns').skew(),
+           ddf.rolling(7, center=True, axis='columns').skew())
```

Comment:
Looks like you are correctly testing the single-partition case.  Remove the comment TODO ?